abuseipdb_connector.py

```diff
@@ -36,6 +37,22 @@ def __init__(self):
         # Call the BaseConnectors init first
         super(AbuseipdbConnector, self).__init__()
 
+    def _is_ip(self, input_ip_address):
+        """Function that checks given address and return True if address is valid IPv4 or IPV6 address.
```

Comment:
I only see this used below in the context of `ipv6`, so which one is it supposed to validate?

Reply:
for ipv4 we already have a validation on platform side, but that doesn't validate ipv6 thats why we have added thia function to also validate ipv6 addresses

Reply:
So why does it say `IPv4 or IPV6` if it only checks for the latter?

Reply:
done